cmd/earthly/build_cmd.go

```diff
@@ -571,11 +572,14 @@ func (app *earthlyApp) actionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs
 				app.logbusSetup.SetOrgAndProject(details.EarthlyOrgName, details.EarthlyProjectName)
 				if doLogstreamUpload {
 					app.logbusSetup.StartLogStreamer(cliCtx.Context, cloudClient)
-					app.console.Printf("Streaming logs to %s\n", logstreamURL)
 				}
 			}
 		}
 	}()
+
+	if app.logstream && doLogstreamUpload {
+		app.console.ColorPrintf(color.New(color.FgHiYellow), "Streaming logs to %s\n\n", logstreamURL)
+	}
```

Comment:
Does this show up under `Init` instead of `Build` now by any chance?

Reply:
yes, I had to remove it from the go routine, I just hope there aren't any significant cases of showing the message without it being available.